### PR TITLE
Change tracking: Add data dictionary entries

### PR DESCRIPTION
**Pull request by rhetoric101**
Data Source: Change Tracking
Data Type: Deployment

Here is the raw list of data from the query builder (I made some minor tweaks to the wording in the dictionary entries):

- changelog - A URL for the changelog or list of changes if not linkable.
- commit - The commit identifier. for example, the Git commit SHA.
- deepLink - A URL back to the system generating the deployment. This field can hold any URL.
- deploymentId - Unique identifier generated by New Relic at the time the marker is created. Numeric identifiers are from the legacy APM Deployment Marker REST API, alphanumeric identifiers are from the current NerdGraph API.
- deploymentType - The type of deployment. One of: 'BASIC', 'CANARY', 'BLUE-GREEN', 'OTHER', 'ROLLING', 'SHADOW'
- description - A description of the deployment.
- entity.guid - The globally unique identifier for the New Relic entity this marker applies to.
- entity.name - The friendly name for the New Relic entity this marker applies to.
- groupId - String value that can be used to correlate multiple entities across a single change event.
- timestamp - The start time of the deployment, in milliseconds since Unix epoch.
- user - Username of the deployer or bot that created this deployment.
- version - The version of the deployment, for example: v1.1.

### CODE REVIEW

File: src/content/docs/change-tracking/change-tracking-view-analyze.mdx [hunk 1]
```diff
@@ -161,6 +161,10 @@ After you create a marker using GraphQL, you can use NRQL in the [query builder]
 
 Try these examples or create your own queries:
 
+<Callout variant="tip">
+  If you need help with the field names and descriptions, take a look at our [data dictionary](/attribute-dictionary/?dataSource=Change+tracking&event=Deployments)
```
Comment (zuluecho9) [inline src/content/docs/change-tracking/change-tracking-view-analyze.mdx:165]:
```suggestion
  For details about the data structure and attribute definitions, see our [data dictionary](/attribute-dictionary/?dataSource=Change+tracking&event=Deployments)
```

File: src/data-dictionary/events/Deployment/groupId.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+name: groupId
```
Comment (zuluecho9) [inline src/data-dictionary/events/Deployment/groupId.md:2]:
```suggestion
name: groupId
units: ID
```

File: src/data-dictionary/events/Deployment/entity.guid.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+---
+name: entity.guid
+type: attribute
```
Comment (zuluecho9) [inline src/data-dictionary/events/Deployment/entity.guid.md:3]:
```suggestion
type: attribute
units: ID
```

**Review by zuluecho9 — APPROVED** [review]

File: src/data-dictionary/events/Deployment/entity.guid.md (hunk 3 above)
Comment (rhetoric101) [reply src/data-dictionary/events/Deployment/entity.guid.md:3]: @zuluecho9 the entity.guid is actually not an ID field. It is a free-form field (according to our SME).

**Pull request merged**